spec/sparsam_spec.rb

```diff
@@ -31,6 +31,12 @@
       subdata.id_s.should == "id_s default"
     end
 
+    it "respects equality of Sets" do
```

Comment:
Don't know too much about the way sparsam's tests are structured, but it seems strange to nest this within `describe Sparsam::Serializer` block

Reply:
good call. I missed this.